### DISCUSSION TITLE
Cannot push local branch

### DISCUSSION THREAD

**Discussion by ariellellouch** [DASDAE/dascore#273] [Q&A]
Hi all,

I'm trying to push a new branch I made (dispersion image computation, yay!). 
Despite following the contribution instructions and using a personal token, I get this error:

remote: Permission to dasdae/dascore.git denied to ariellellouch.
fatal: unable to access 'https://github.com/dasdae/dascore/': The requested URL returned error: 403

Is there any way I need to be added as a contributor/collaborator to be allowed to push, or is it a dumb problem on my side?

Thanks!
Ariel

**Comment by d-chambers** [comment]
Hey @ariellellouch,

Yes, you either need to use your personal fork or be in the DASDAE developer group. I will add you to the group and make an issue so we remember to update the documentation for future contributors. 

I am looking forward to seeing the PR!

**Comment by ariellellouch** [comment]
Sweet, it works now (still need to adhere to the todo list, but will try to do it soon) !
